lexos/helpers/definitions.py

```diff
@@ -5,3 +5,20 @@
 _WORD_BOUNDARY_REGEX_STR = r'\s'
 
 WORD_REGEX = re.compile(_WORD_REGEX_STR, re.UNICODE)
+
+
+def count_phrase_in_text(phrase: str, text: str):
+    """Count how many times the phrase appears in the text
+
+    :param phrase: string containing a phase or word
+    :param text: string where the phrase will get count
+    :return: count: number of times the phrase appears in the text
+    """
+    count = 0
+    if text.startswith(phrase + " "):
+        count += 1
+    if text.endswith(" " + phrase + "\n") or text.endswith(" " + phrase) or \
```

Comment:
I think it would be better if you can handle `"\r\n"`, although I think lexos currently converts line-ending on upload, I think it is nice to build more robust functions.

Reply:
I can do that, but isn't redundant if lexos already handles it?

Reply:
robustness, always about robustness